tasks/main.yml

```diff
@@ -44,6 +52,14 @@
   environment: project_environment
   when: project_has_npm
 
+- name: Check if bower dir exists
```

Comment:
Checking if a previous install exists isn't needed when `project_copy_previous_bower_components` is off, so maybe add a `when` to this task?